ui-spacetimechart/src/types.ts

```diff
@@ -0,0 +1,15 @@
+export type SpaceTimeRegion = {
+  type?: string;
+  timeStart: number; // timestamp
+  timeEnd: number; // timestamp
+  spaceStart: number; // mm
+  spaceEnd: number; // mm
+  image?: string;
+};
+
+export type WorkSchedule = {
+  type: 'TRACK' | 'CATENARY';
+  timeStart: string;
+  timeEnd: string;
```

Comment:
Could we define these as `Date` instead of `string`?

Reply:
done